client/src/css/ui-grid.css

```diff
@@ -57,3 +57,20 @@
   /* this is full height minus the header crumb and util bar */
   height : calc(100vh - 65px - 35px);
 }
+
+.error {
```

Comment:
It is preferable to use the bootstrap classes and/or colors to keep uniform style.  Are those classes insufficient?

If you must create your own classes, it is clearer to prefix these classes with the component they affect, like bootstrap (e.g `bg-danger`, `btn-primary`, etc).  Could you put a prefix on these?